lib/link_checker/uri_checker/http_checker.rb

```diff
@@ -291,6 +295,8 @@ def additional_connection_headers
     end
 
     def use_google_safebrowsing?
+      return false if gov_uk_uri? && !gov_uk_upload_uri?
```

Comment:
We will probably need to check against URLs like this too:

```
https://assets.publishing.service.gov.uk/media/...
```

Reply:
I had tried to look through a few different links on whitehall to figure out any other urls I would need to check but couldn't really spot any other obvious ones. Do you think the "/government/uploads" and the one you've just mentioned will be the only two needing to be checked?

Reply:
I think so, yeah. The `/government/uploads` URLs come from the Whitehall attachments system, and the ones I just mentioned come from `asset-manager`.

Reply:
👍 ok, cool I'll add it in as well then

Reply:
Hmm, @thomasleese I just added an extra test and realised that I don't actually need to modify the code to include this check because the way it is written at the moment it doesn't consider ```https://assets.publishing.service.gov.uk/media/``` to be a `gov_uk_uri` and therefore already sends it for a google safebrowsing request. Do you want me to make it more explicit or clear that it checks these urls or is it ok as it is?

Reply:
I reckon if we can just add a test to make sure it's not being checked and that should be enough.

Reply:
👍 added the extra test @thomasleese